data/i18n/pt/pt.toml

```diff
@@ -0,0 +1,225 @@
+ # i18n strings for the Portuguese (main) site.
```

Comment:
maybe in the portuguese localization it is worth to already use pt-br instead of pt so we can merge this together with the #27413 

Even if we don't merge this together, at least the destination file will have the correct name :)